### PR TITLE
add 'new entity' to context menu

### LINKED ISSUE #1153: DMT: "New blueprint" and "New Entity" in context menu
Should be added as class methods on the TreeNode

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- Adds a new method to TreeNode - "addEntity()"
- New context menu entries for packages - "New entity" and "New Blueprint"

## Issues related to this change:

closes #1153

### CODE REVIEW

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx [hunk 1]
```diff
@@ -261,6 +275,133 @@ export const NodeRightClickMenu = (props: {
...
+                <Label label={'Blueprint'} />
+                <BlueprintPicker
+                  onChange={(selectedType: string) =>
+                    setFormData({ type: selectedType })
+                  }
+                  formData={formData?.type || ''}
+                />
+              </div>
+              {loading ? (
+                <Button
```
Comment (eoaksnes) [inline web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:302]: Should the button be there?

**Review by eoaksnes — APPROVED** [review]

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx (hunk 1 above)
Comment (soofstad) [reply web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:302]: Yes, its a button with a loading icon

Comment (eoaksnes) [reply web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:302]: Roger :)

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx [hunk 2]
```diff
@@ -261,6 +275,133 @@ export const NodeRightClickMenu = (props: {
...
+                    width: '200px',
+                    alignSelf: 'center',
+                  }}
+                >
+                  <Progress.Dots />
+                </Button>
+              ) : (
+                <Button
+                  disabled={formData?.type === undefined}
+                  style={{
```
Comment (KristianKjerstad) [inline web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:314]: looks like this style is repeated quite a few times.

Maybe its possible to used a styled component?
(I'm not sure if this is supported with the EDS Button though...)

**Review by KristianKjerstad — APPROVED** [review]
good!

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx (hunk 2 above)
Comment (soofstad) [reply web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx]: Could not use styled-components, but used a config object :)

**Pull request merged**